src/test/java/ch/jalu/configme/beanmapper/propertydescription/BeanDescriptionFactoryImplTest.java

```diff
@@ -134,34 +134,17 @@ void shouldThrowForMultiplePropertiesWithSameName() {
     void shouldThrowForWhenExportNameIsNullForProperty() {
         // given / when
         ConfigMeMapperException ex = assertThrows(ConfigMeMapperException.class,
-            () -> factory.getAllProperties(BeanWithEmptyName.class));
+            () -> factory.collectProperties(BeanWithEmptyName.class));
 
         // then
         assertThat(ex.getMessage(),
-            equalTo("Custom name of Bean property '' with getter 'public java.lang.String ch.jalu.configme.samples.beanannotations.BeanWithEmptyName.getAuthor()' may not be empty"));
-    }
-
-    @Test
-    void shouldReturnCommentsWithUuidIfNotRepeatable() {
```

Comment:
Intentionally removed?

Reply:
moved as BeanInstantiationServiceImplTest#shouldCacheInstantiations

It's important that instantiations be cached, otherwise unique comments are broken. This is not very cool. :/

Reply:
#454